recipes/serf/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "serf" %}
+{% set version = "1.3.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.bz2
+  url: https://archive.apache.org/dist/{{ name }}/{{ name }}-{{ version }}.tar.bz2
+  sha256: 549c2d21c577a8a9c0450facb5cca809f26591f048e466552240947bdf7a87cc
+  patches:
+    - SConstruct.diff
+
+build:
+  number: 0
+  run_exports:
+    # https://abi-laboratory.pro/tracker/timeline/serf/
+    # They keep changing soname b/w patch versions :'(
+    - {{ pin_subpackage('serf', max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ compiler("c") }}
+    - cmake                # [win]
+    - scons
+  host:
+    - apr
+    - expat
+    - krb5
+    - openssl
+    - zlib
+  run:
+    - apr
+    - expat
+    - krb5
+    - openssl
+    - zlib
+
+test:
+  commands:
+    - test -f "${PREFIX}/lib/libserf-1.a"                          # [unix]
+    - test -f "${PREFIX}/lib/libserf-1${SHLIB_EXT}"                # [unix]
+    - if not exist %PREFIX%\\Library\\lib\\serf-1.lib exit 1       # [win]
+    - if not exist %PREFIX%\\Library\\bin\\libserf-1.dll exit 1    # [win]
+    - if not exist %PREFIX%\\Library\\lib\\libserf-1.lib exit 1    # [win]
+
+about:
+  home: http://serf.apache.org
+  license: Apache 2.0
+  license_family: Apache
+  license_file: LICENSE
+  summary: High performance C-based HTTP client library
+  description: |
+    Serf is a simple web server. It is a development tool to be used during the
+    development of rich internet applications.It is not intended for deployment.
+  dev_url: https://svn.apache.org/repos/asf/serf/
+
+extra:
+  recipe-maintainers:
+    - jakirkham
+    - mingwandroid
+    - nehaljwani
```

Comment:
Have gone ahead and added @mingwandroid and @nehaljwani as they authored the recipe in `defaults`. Please let me know if that is ok.

Reply:
I am fine with it.